ocpmodels/trainers/energy_trainer.py

```diff
@@ -250,7 +253,7 @@ def train(self):
                         split="train",
                     )
 
-            self.scheduler.step()
+            self.scheduler.step(epoch)
```

Comment:
`epoch` would initially be 0 (if not starting from a checkpoint). Would this step correctly in that case? Or should this instead be `self.scheduler.step(epoch+1)`?